between random: 232818981
between random: 232818981
between random: 232818974
between random: 232818981
between random: 232818981
between random: 16554
between random: [218,61,137,100,6,243,57,206]
between random: [177,193,96,238,32,12,63,61]
between random: [177,193,96,238,32,12,63,61]
between random: [177,193,96,238,32,12,63,61]
between random: [177,193,96,238,32,12,63,61]
between random: [177,193,96,238,32,12,63,61]
between random: [177,193,96,238,32,12,63,61]
between random: [177,193,96,238,32,12,63,61]
between random: [177,193,96,238,32,12,63,61]
between random: [177,193,96,238,32,12,63,61]
between random: [230,139,235,10,81,34,112,94]
between random: [230,139,235,10,81,34,112,94]
between random: [230,139,235,10,81,34,112,94]
between random: [230,139,235,10,81,34,112,94]
between random: [166,136,71,158,41,116,254,131]
between random: [166,136,71,158,41,116,254,131]
"    require ~sent_pkt(scid,pkt.ptype,pkt.seq_num);  # [4]
"
/usr/local/lib/python2.7/dist-packages/ivy/include/1.7/quic_packet.ivy: line 397: error: assumption failed
output directory: temp/222
implementation directory: /home/chris/TVOQE_UPGRADE_27/quic/quic-go/server/
implementation command: ./server -p 4443 127.0.0.1
../quic_server_test_newcoid_rtp_error (0) ...
implementation command: ['./server', '-p', '4443', '127.0.0.1']
quic_process pid: 8126
timeout 100 ./build/quic_server_test_newcoid_rtp_error seed=98 the_cid=0 server_cid=1 client_port=4987 client_port_alt=4988
client return code: 1
FAIL
error: 1 tests(s) failed
